scripts/salt-setup.sh

```diff
@@ -46,6 +46,53 @@ function apply_optional_states {
   fi
 }
 
+# This adds entities to the 'roles' key to be used during actual cluster install. Hence, this does not use the temporary config location.
+function add_single_role_for_cluster_salt {
+  local role=${1}
+  ${SALT_PATH}/bin/salt-call --local grains.append roles ${role} -l info --log-file=/tmp/salt-build-${saltenv}.log --log-file-level=debug
+}
+
+function add_prewarmed_roles {
+  if [ "${INCLUDE_FLUENT}" == "Yes" ]; then
+    # Note: This will need to be changed if making changes to versions etc in the prewarmed image.
+    local fluent_prewarmed="fluent_prewarmed_v1"
```

Comment:
can we het this value somehow differently from a config or something? it's a bit hidden in the code

Reply:
Moved it to the Makefile